src/formatDate.js

```diff
@@ -50,7 +50,31 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const splittedData = date.split(fromFormat[3]);
+  const arrayFrom = [[fromFormat[0], splittedData[0]],
+    /*             */[fromFormat[1], splittedData[1]],
+    /*             */[fromFormat[2], splittedData[2]]];
```

Comment:
так, ненужно ничего выдумывать, все уже выдумали)
```suggestion
  const arrayFrom = [
    [fromFormat[0], splittedData[0]],
    [fromFormat[1], splittedData[1]],
    [fromFormat[2], splittedData[2]],
  ];
```

Reply:
[[kay, property]] - entries
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Object/entries

Reply:
> так, ненужно ничего выдумывать, все уже выдумали)

Ловко, возьму на вооружение!!!